libtiledbsoma/src/soma/soma_array.h

```diff
@@ -598,6 +598,17 @@ class SOMAArray : public SOMAObject {
             ctx_->tiledb_ctx(), arr_);
     }
 
+    /**
+     * @brief Get members of the schema (capacity, allows_duplicates,
+     * tile_order, cell_order, offsets_filters, validity_filters, attr filters,
+     * and dim filters) in the form of a PlatformConfig
+     *
+     * @return PlatformConfig
+     */
+    PlatformConfig config_options() const {
```

Comment:
This sounds passive & it's not clear from the name alone who's doing what ... maybe `read_back_config_options` or `get_config_options` or `config_options_from_schema` (I like the last one best) ...